### PR TITLE
Support for extension parameters

### PR DESCRIPTION
**Pull request by jelz**
CEP: https://contentful.atlassian.net/wiki/spaces/ENG/pages/397574329/CEP-0048+Configuration+for+UI+Extensions
API change: https://github.com/contentful/content_api/pull/1191

### CODE REVIEW

File: docs/extension/create/README.md [hunk 1]
```diff
@@ -8,35 +8,38 @@ Creates an extension
...
+  --field-types              Field types                 [array] [required]
+  --descriptor               Path of descriptor file               [string]
+  --src                      Src                                   [string]
+  --srcdoc                   Path of srcdoc file                   [string]
+  --sidebar                  Renders in sidebar                   [boolean]
+  --installation-parameters  Installation parameter values         [string]
 
 One of src or srcdoc is required
 ID, name, field types, src, srcdoc and sidebar arguments can also be provided via the descriptor file
+Parameters can be defined only in the descriptor file
```
Comment (floelhoeffel) [inline docs/extension/create/README.md]: I'd rephrase:

"Installation parameters need to be defined in the descriptor file."
"Values for `installation parameters` can only be provided as CLI arguments."

File: lib/cmds/extension_cmds/update.js [hunk 2]
```diff
@@ -41,6 +41,10 @@ export const builder = (yargs) => {
       type: 'boolean',
       describe: 'Force operation without explicit version'
     })
+    .option('installation-parameters', {
+      type: 'string',
+      describe: 'JSON string of installation parameter values'
```
Comment (floelhoeffel) [inline lib/cmds/extension_cmds/update.js]: maybe:

JSON string of installation parameter key-value pairs

?

File: docs/extension/create/README.md [hunk 3]
```diff
@@ -8,35 +8,38 @@ Creates an extension
...
-  --sidebar      Sidebar                                          [boolean]
+  --id                       ID                                    [string]
+  --name                     Name                       [string] [required]
+  --space-id                 Space ID                   [string] [required]
+  --field-types              Field types                 [array] [required]
+  --descriptor               Path of descriptor file               [string]
+  --src                      Src                                   [string]
+  --srcdoc                   Path of srcdoc file                   [string]
+  --sidebar                  Renders in sidebar                   [boolean]
+  --installation-parameters  Installation parameter values         [string]
```
Comment (floelhoeffel) [inline docs/extension/create/README.md]: key-values rather than values?

File: test/integration/cmds/extension/snapshots/create.test.js.md [hunk 4]
```diff
@@ -11,14 +11,16 @@ Generated by [AVA](https://ava.li).
...
+      -h, --help                 Show help                                 [boolean]␊
+      --id                       Extension id                               [string]␊
+      --name                     Extension name                             [string]␊
+      --space-id                 Space id                                   [string]␊
+      --field-types              Field types                                 [array]␊
+      --descriptor               Path to an extension descriptor file       [string]␊
+      --src                      URL to extension bundle                    [string]␊
+      --srcdoc                   Path to extension bundle                   [string]␊
+      --sidebar                  Render the extension in the sidebar       [boolean]␊
+      --installation-parameters  JSON string of installation parameter values␊
```
Comment (floelhoeffel) [inline test/integration/cmds/extension/snapshots/create.test.js.md]: if you say this is a JSON string, then also do that in the above examples where its just a string

File: docs/extension/create/README.md (hunk 3 above)
Comment (jelz) [reply docs/extension/create/README.md]: There's an exmaple below and we say "key-value pairs" in the output. It's already the longest line.
